### DISCUSSION TITLE
win_start can't start successfully

### DISCUSSION THREAD

**Discussion by HysTakuya** [Sanster/IOPaint#211] [Q&A]
lama-cleaner can't start successfully as following screenshot
I've tried to delete "clickseg_pplnet.pt" and restart but the problem still exists.
Please kindly guide me if you know how to fix it, thanks!
![win_start_issue](https://user-images.githubusercontent.com/124450072/218486772-08bee555-82de-4cad-b4c9-21c942ce1acb.png)

**Comment by Sanster** [comment]
Please download `clickseg_pplnet.pt` model from [here](https://github.com/Sanster/models/releases/tag/clickseg_pplnet), and put it in the model directory
<img width="852" alt="image" src="https://user-images.githubusercontent.com/3998421/218612812-75e95255-6c3d-4b29-8c7e-6abd0bf8b1c2.png">

**Reply by HysTakuya** [reply]
Very thanks for your reply! But the problem still seems to happen
![未命名3](https://user-images.githubusercontent.com/124450072/220285491-e7f64a7c-81bf-4fcd-a5c6-fbec607d499f.png)

**Reply by Sanster** [reply]
Sorry about that, other users have also encountered this problem, if I can reproduce it and solve it, I will reply in this issue https://github.com/Sanster/lama-cleaner/issues/213
